apps/api/src/common/config/schemas.ts

```diff
@@ -23,6 +23,7 @@ const baseSchema = z.object({
     BRIDGE_HUB_ADDRESS: addressSchema,
     SHARED_BRIDGE_ADDRESS: addressSchema,
     STATE_MANAGER_ADDRESSES: addressArraySchema,
+    ENVIRONMENT: z.enum(["mainnet", "testnet", "local"]).default("mainnet"),
```

Comment:
Might want to indicate that `mainnet` is the default somewhere in the docs? (.env.example, README, etc)

Reply:
will do it in another PR where I attack all documentation and read me detailed 🫡